### DISCUSSION TITLE
Sequelize v7 Stable release roadmap

### DISCUSSION THREAD

**Discussion by vinodg006** [sequelize/sequelize#15866] [Help & Questions] [answered]
When is the team planning to release LTS for sequelize v7, is there any ETA for this? Also the dependent packages like sequelize-cli, forest-express-sequelize would need upgrade?

**Comment by ephys** [comment, marked as answer]
Please see https://github.com/sequelize/sequelize/discussions/15736

It is very likely that all sequelize related packages will need to update, but some may continue to work
